prow/jobs/kyma/tests/test-logging-monitoring/test-logging-monitoring.yaml

```diff
@@ -1,14 +1,14 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
 job_template: &job_template
+  name: kyma-tests-test-logging-monitoring
   skip_report: true
-  run_if_changed: "^tests/test-logging-monitoring/"
   decorate: true
   path_alias: github.com/kyma-project/kyma
   max_concurrency: 10
-  extra_refs:
-    - org: kyma-project
-      repo: test-infra
-      base_ref: master
-      path_alias: github.com/kyma-project/test-infra
   spec:
     containers:
       - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd
```

Comment:
Can you please also add resource limitations as in [here](https://github.com/kyma-project/test-infra/blob/master/prow/jobs/bundles/bundles.yaml#L19)?

Reply:
done.